src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,49 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void sizeTest() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
컬렉션 검증 함수를 활용해 보시는 것도 추천드립니다 😄 
```suggestion
        assertThat(numbers).hasSize(3);
```